src/components/Post/Comment/EditableReplyContent.tsx

```diff
@@ -48,11 +53,19 @@ const EditableReplyContent = ({ userId, className, commentId, content, replyId }
 	// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
+	useEffect(() => {
+		replyToreply.setFieldValue('content', `[@${userName}](${global.window.location.origin}/user/${userName})` || '');
```

Comment:
this useEffect has no dependency array, it'll trigger an infinite loop

Reply:
added empty array as dependency